modules/access_level/versions.tf

```diff
@@ -20,7 +20,7 @@ terraform {
 
     google = {
       source  = "hashicorp/google"
-      version = ">= 3.50, < 5.0"
+      version = "~> 4.0"
```

Comment:
We should continue to support `>= 3.50, < 5.0` for this module and others, since we are no depending on any 4.X specific features.